core-plugins/docs/Email-postaction.md

```diff
@@ -0,0 +1,57 @@
+# Email Post Action
+
+
+Description
+-----------
+Sends an email at the end of a pipeline run.
+
+
+Use Case
+--------
+This action can be used when you want to send an email at the end of a pipeline run.
+For example, you may want to configure a pipeline so that an email is sent whenever
+the run failed for any reason.
+
+
+Properties
+----------
+**recipientEmailAddress:** The address to send the email to.
+
+**senderEmailAddress:** The address to send the email from.
+
+**message:** The message of the email.
```

Comment:
if we allow other types, then the message can't be a string.  I think its best to keep it simple for now until we have a use case for more complex things.